contracts/AckToEarn.sol

```diff
@@ -24,6 +24,7 @@ contract AckToEarn is Ownable, ReentrancyGuard {
         address recipient;
         string responseEmailAddress;
         string message;
+        string fileCid;
```

Comment:
Should this be an array of strings, in case multiple files are uploaded?

Reply:
hm let me check

Reply:
i believe filecoin just makes one folder and gives you a `fileCid` with that folder's id but worth double checking

Reply:
i can only choose one file on my machine